.gitignore

```diff
@@ -23,7 +23,7 @@ liboctave.*
 liboctinterp.*
 *.pyc
 __pycache__
-*.so
+PsychPython/psychtoolbox/*.so
```

Comment:
This is better, but what confuses me is why you think it is needed at all? I don't remember any .so's being stored there during a run of setup.py?

Reply:
If you do install, probably not because they end up living in the build directory. But the editable installation mode (setup.py develop or pip -e) builds and stores them in place I'm there directory. In an editable install, when you import psychtoolbox, it's actually just importing via a symlink from the git PsychPython/pychtoolbox directory (hence why the import failed when the library files lived in a directory not named psychtoolbpx, like in master).

Reply:
(If this does not make sense, I can elaborate later when I'm not on mobile)